modules/standard/FileSystem.chpl

```diff
@@ -21,12 +21,12 @@
 
    The FileSystem module focuses on file and directory properties and
    operations.  It does not cover every interaction involving a file -
-   for instance, path specific operations live in the :mod:`Path` module, while
-   opening, writing to, or reading from a file appropriately live in the
+   for instance, path-specific operations live in the :mod:`Path` module, while
+   opening, writing to, or reading from a file live in the
    :mod:`IO` module.  Rather, it covers cases where the user would prefer a file
-   or directory handled wholesale and/or with minimal interaction.  The
-   :proc:`copy` functionality belongs here, as does :proc:`chmod`,
-   :proc:`mkdir`, and :proc:`remove`, for instance.  Also included are
+   or directory handled wholesale and/or with minimal interaction.  For example,
+   this module contains file :proc:`copy` functionality, as well as :proc:`chmod`,
```

Comment:
I like that better, but think I'm balking at the use of "the" because it suggests that there could never be another routine that provides this functionality (i.e., that it's a singleton/unique).  I started running with "a file copy routine", but then that led to additional changes in the nexte sentence and earlier in the paragraph and so on, which overall I liked pretty well.  Hoped to have you review before committing, but I think you've left for the evening.  Given that you seemed OK with the code changes, I'm going to commit what I have now for tonight's testing, but with the intention that we can continue to play with this further tomorrow/up to doc freeze.

Related, and occurring to me as I wrestled with this paragraph:  For long modules like this, I'm wondering if we want to have this top-level comment give a "directory" of all routines in the module so that rather than "for instance"-style sentences we have a complete list of all routines broken into various categories, serving as an index of sorts.  That said, I don't know what those categories might be other than "filerators"...  Do you have a mental map like this?